packages/experimental/Button/src/Button.types.ts

```diff
@@ -111,16 +111,62 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
   fluid?: boolean;
 }
 
+export interface ButtonMacProps {
+  /*
+   * Name of the icon to show on the Button.
+   */
+  image?: Image;
+  /*
+   * Text to show on the Button.
+   */
+  title?: string;
+  /*
+   * ButtonStyle wrapped enum
+   */
+  buttonStyle?: ButtonMacStyle;
+  /*
+   * A button can be enabled or disabled.
+   */
+  isEnabled?: boolean;
+  /*
+   * A callback to call on button click event
+   */
+  onPress?: () => void;
+}
+
+export interface ButtonMacTokens {
+  /*
+   * Button content tint color
+   */
+  contentTintColor?: number;
+  /*
+   * Button background color for rest state
+   */
+  restBackgroundColor?: number;
+}
```

Comment:
I wonder if instead of defining these here, we could make a `ButtonTokens.macos.ts` and just import `ButtonTokens` and keep the platform specific tokens separate that way

Reply:
Now that we have a component for native button, all platforms should conform to the same interface.